src/internal/base-component/metrics/metrics.ts

```diff
@@ -78,7 +79,7 @@ export class Metrics {
    * component was loaded. The component load event will only be reported as used to client logging
    * service once per page view.
    */
-  logComponentLoaded() {
+  logComponentsLoaded() {
```

Comment:
Renamed the function, because it has nothing to do with a particular component, it does not even know its name. This call reports the entire bundle being loaded.